### PR TITLE
feat: update home page with initiative links, add back button to individual submission page

### PR DESCRIPTION
**Pull request by wilwong89**
# Description
Remove placeholder Vue links
Directly display initiatives selection on the homepage, with greyed out button when not logged in.
Added back button to individual submissions to go back to submissions list page

Files changed:
Navbar.vue
HomeView.vue
SubmissionVue.vue

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/views/HomeView.vue [hunk 1]
```diff
@@ -56,35 +27,14 @@ const languagesEcosystem: Array<{ text: string; href: string }> = [
 
   <div class="text-center">
     <h1 class="font-bold">Welcome to the PermitConnect Navigator Service!</h1>
```
Comment (kyle1morel) [inline frontend/src/views/HomeView.vue:29]: Text does not match ticket.

**Comment by kyle1morel** [comment]
I would double check with Tyler about the UI for the home view. The ticket calls for the initiative text and button to be centered, but I think that may look odd.

**Comment by tpantella** [comment]
Please center "Choose an Initiative" and have the button centered - for now this change will be visually pleasing and appropriate, though as the content grows we may want to reconsider left alignment in the future.

Otherwise looks good.

File: frontend/src/views/HomeView.vue (hunk 1 above)
Comment (wilwong89) [reply frontend/src/views/HomeView.vue]: Fixed with text as per ticket.

**Review by kyle1morel — APPROVED** [review]

**Pull request merged**